pyuvdata/uvdata/mwa_corr_fits.py

```diff
@@ -315,9 +315,16 @@ def correct_cable_length(self, cable_lens):
         A list of strings containing the cable lengths for each antenna.
 
         """
+        # as of version 0.29.X cython does not handle numpy arrays of strings
+        # particularly efficiently. Casting to bytes, then into this demonic
+        # form is a workaround found here: https://stackoverflow.com/a/28777163
+        cable_lens = np.asarray(cable_lens).astype(np.string_)
+        cable_lens = cable_lens.view("uint8").reshape(
+            cable_lens.size, cable_lens.dtype.itemsize
+        )
```

Comment:
silly question: presumably to do any work on these things they have to be in numeric types. I'm not sure I understand why they are strings, but assuming there's a good reason (units?) could we convert to the numeric types before calling the cython?

Reply:
So I've printed this out before, and you get entries like "EL_-756332" and "-33745" random numbers i picked. But it appears as though _some_ entries have this "EL_" prepended which implies the physical length of the cable has already had the electric velocity factor of 1.204 applied which others do not. We very much can do the parsing and conversion in python, this was just the solution I found that let me do it in Cython.

Reply:
It would be a lot easier if the metadata only had a single type of cable length and not a possible mix of both. Then, like you said, it could just be stored as a numeric array.